charts/maildev/values.yaml

```diff
@@ -53,11 +54,15 @@ cron:
 
 ingress:
   enabled: false
+  spec:
+  # ingressClassName is standard for 1.19+
+    ingressClassName: nginx  
```

Comment:
Can you set this values to empty by default ?

Reply:
And also done, hmmm how do I progress this..